app/controllers/idv/confirmations_controller.rb

```diff
@@ -71,8 +65,6 @@ def add_proofing_component
 
     def finish_idv_session
       @code = personal_key
-      user_session[:personal_key] = @code
-      idv_session.personal_key = nil
```

Comment:
it seemed like `Idv::Session#personal_key` was being written to a bunch, when the session was right there? So I just decided to remove it to simplify, since we already store the data across requests in `user_session`